src/main/java/decodex/storage/Storage.java

```diff
@@ -1,5 +1,114 @@
 package decodex.storage;
 
+import decodex.ui.messages.ErrorMessages;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Scanner;
+
 public class Storage {
 
+    /**
+     * Variables for loading from the default input filepath.
+     */
+    public static final String DEFAULT_INPUT_DIRECTORY = "input";
+
+    /**
+     * Variable for the default output directory.
+     */
+    public static final String DEFAULT_OUTPUT_DIRECTORY = "output";
+
+    /**
+     * Initializes a new Storage.
+     */
+    public Storage() throws IOException {
+        instantiateDirectories();
+    }
+
+    /**
+     * Reads the contents from the default input file.
+     *
+     * @param fileName The name of the file specified by the user.
+     * @return The contents from the default input file.
+     * @throws IOException If the default input file does not exist.
+     */
+    public String readFromDefaultInputFile(String fileName) throws FileNotFoundException {
+        String inputContent = "";
+        File inputDirectory = new File(DEFAULT_INPUT_DIRECTORY);
+        File inputFile = new File(inputDirectory, fileName);
+
+        try {
+            Scanner in = new Scanner(inputFile);
+            while (in.hasNext()) {
+                String fileLine = in.nextLine();
+                inputContent = inputContent + fileLine;
+            }
+        } catch (FileNotFoundException e) {
+            throw new FileNotFoundException(ErrorMessages.DEFAULT_INPUT_FILE_DOES_NOT_EXIST_MESSAGE);
+        }

```

Comment:
There are definitely going to be more errors, like permissions, repeated file name. Perhaps just handle filenotfoundexception with a special message, then every other generic file/io related exception as a general exception?